label_studio/projects/api.py

```diff
@@ -645,7 +645,9 @@ class ProjectSampleTask(generics.RetrieveAPIView):
     serializer_class = ProjectSerializer
     swagger_schema = None
 
-    def retrieve(self, request, *args, **kwargs):
-        config = request.GET.get('label_config')
+    def post(self, request, *args, **kwargs):
+        json_data = json.loads(request.body)
```

Comment:
same here as above

Reply:
I'll fix it in release/1.0.1 branch after merge